smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -290,6 +291,33 @@ void restTemplateClient() {
                             .hasAttribute(HttpAttributes.HTTP_ROUTE, "/ping")));
   }
 
+  @Test
+  @org.junit.jupiter.api.Order(3)
+  void webflux() {
+    resetExporters();
```

Comment:
perhaps move this to a `@BeforeEach` method

Reply:
this doesn't work, because one of the test methods asserts the telemetry data from app startup.

Reply:
I guess it is fine. Alternatively could use a boolean that is set in the test that needs the startup info to enable reseting state.

Reply:
not sure how that could work